src/components/dashboard/indicators/IndiceUv.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+
+import useIndicators from 'hooks/useIndicators'
+import MagicLink from 'components/base/MagicLink'
+import Card from 'components/misc/Card'
+import Chart from './indiceUv/Chart'
+
+export default function IndiceUv(props) {
+  const { data, isError, isLoading } = useIndicators(props.place.code)
+
+  return (
+    <Card columns={6}>
+      <Card.Content>
+        <Card.Header>
+          <Card.Info>
+            <Card.Title isLoading={isLoading} indicateur='indice_uv'>
+              Indice UV
+            </Card.Title>
+            <Card.Value isError={isError}>
+              {isError
+                ? 'Zut 🦙'
+                : data &&
+                  (data.indice_uv?.indice?.label
+                    ? data.indice_uv.indice.label
+                    : 'Pas de données')}
```

Comment:
Je suis pas calé calé en JS, mais tu ne peux pas faire un truc comme ça ?
```suggestion
                    data?.indice_uv?.indice?.label? ?? 'Pas de données')}
```